docs/how-to-use-relational-backend-storage.md

```diff
@@ -87,6 +91,81 @@ Finally, you can run the script in the distribution package directory to start t
 ./${GRAVITINO_HOME}/bin/gravitino.sh start
 ```
 
-## How to use H2
+## How to use PostgreSQL
+
+### Prerequisites
+
+- PostgreSQL 12 ~ 16. For other versions, the community has not tested fully and may not be compatible.
+- Gravitino distribution package with version `0.6.0` or above.
+- PostgreSQL connector Jar (Should be compatible with the version of PostgreSQL instance).
+
+### Step 1: Get the initialization script
+
+You need to `download` and `unzip` the distribution package firstly, please see
+[How to install Gravitino](how-to-install.md).
+
+Then you can get the initialization script in the directory:
+
+```text
+${GRAVITINO_HOME}/scripts/postgresql/
+```
+
+The script name is like `schema-{version}-postgresql.sql`, and the `version` depends on your Gravitino version.
+For example, if your Gravitino version is the latest release version, then you can choose the **latest version** script.
+If you used a legacy script, you can use `upgrade-{old version}-to-{new version}-postgresql.sql` to upgrade the schema.
+
+### Step 2: Initialize the database
+
+Please create a database and scheam in PostgreSQL in advance, and execute the initialization script obtained above in the database.
+
+```
```

Comment:
Should add a `shell` or `text` tag here?

Reply:
Ok, it's better to do so.